content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-kubernetes-faq.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Chainguard Enforce for Kubernetes FAQs"
+type: "article"
+description: "Frequently asked questions about Chainguard Enforce for Kubernetes"
+date: 2022-10-18T08:49:31+00:00
```

Comment:
It looks like it should be year-month-day

https://getdoks.org/docs/tutorial/add-pages/#indexmd

```suggestion
date: 2022-11-14T08:49:31+00:00
```